src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -475,6 +467,11 @@ def write_info_json(output_path, parsed_question):
         auto_tags.append("multi_part")
     auto_tags.extend(list(set(q_types)))
 
+    if (difficulties := parsed_question["header"].get("difficulty", ["undefined"])) != ["undefined"]:
+        if not isinstance(difficulties, list):
+            difficulties = [difficulties]
+        auto_tags.extend(f"difficulty-{difficulty.lower()}" for difficulty in difficulties)
```

Comment:
```suggestion
        auto_tags.extend(f"{difficulty.lower()}" for difficulty in difficulties)
```

Thanks for submitting this enhancement ! I'm in favour but I think the difficulty- is redundant and is unnecessary/redundant. It'll also not look great in PL with the tags front end.

Reply:
Fair enough! If we're removing the prefix, we don't need the f-string at all:
```suggestion
        auto_tags.extend(difficulty.lower() for difficulty in difficulties)
```